### ISSUE TITLE
#modxbughunt Moving tvs to modx-resource-main-left/right - Styling issues

### ISSUE THREAD

**Issue by pepebe** [modxcms/revolution#13318]
### Summary
Moving a TV to the first create/update resource tab

modx-resource-main-left
or
modx-resource-main-right

results in a wrong padding to the left.

![screenshot 2017-03-03 10 52 55](https://cloud.githubusercontent.com/assets/667315/23546395/a438c73c-ffff-11e6-9ebe-248c56efdaa9.png)

### Step to reproduce
Move a tv to one of these regions.

### Observed behavior

15 px padding to the left because auf this rule:

````
.x-form-item.modx-tv {
    padding: 0 0 0 15px !important;
}
````
### Expected behavior
No padding to the left.

![screenshot 2017-03-03 10 54 22](https://cloud.githubusercontent.com/assets/667315/23546434/d6356074-ffff-11e6-864f-90ab400f77bd.png)

````
div#modx-resource-main-columns .modx-tv {
    padding-left: 0!important;
}
````

I'll post the pr for this in a few minutes.

### Environment
Any Modx invironment. Its a css issue

**Comment by pepebe** [comment]
I've found this in manager/templates/default/css/index.css, line 4694:

````
  .x-form-item.modx-tv {
    padding: 0 0 0 15px !important;
    /* account for the tv reset icon */ }
````

The original cause for this seems to be the fact, that the reset TV link is displayed like this:

![screenshot 2017-03-03 11 12 57](https://cloud.githubusercontent.com/assets/667315/23547183/96b69582-0002-11e7-80c5-31aea427d7a0.png)

How about we move it to the right?

![screenshot 2017-03-03 11 13 37](https://cloud.githubusercontent.com/assets/667315/23547189/9ff7efe2-0002-11e7-8d83-4c7aba538ee7.png)

manager/templates/default/css/index.css ,line 4659

````
.x-form-item label.x-form-item-label .modx-tv-reset {
    cursor: pointer;
    display: inline-block;
    /* float: left; */
    height: 16px;
    opacity: 0;
    filter: alpha(opacity=0);
    padding: 17px 8px 0 0;
    position: absolute;
    top: 0;
    /*left: -8px;*/
    right: 0;
    transition: all 0.25s;
    width: 16px;
}
````

**Comment by ajilderda** [comment]
This issue has bugged me for a while. :+1: I like positioning the reset button to the right, but prefer it to the right of the label like so:
<img width="747" alt="screen shot 2017-03-03 at 11 39 28" src="https://cloud.githubusercontent.com/assets/487182/23548104/4596d488-0006-11e7-9654-2f16ab366636.png">

```
.x-form-item.modx-tv {
  padding-left: 0 !important;
}
.modx-tv-reset {
  position: relative;
}
.x-form-item label.x-form-item-label .modx-tv-reset {
  position: relative;
  padding: 0;
  
}
.x-form-item label.x-form-item-label .modx-tv-reset:before {
  position: relative;
  padding-left: 4px;
}
```

**Comment by rthrash** [comment]
In prefer the second version next to the label itself. Good job to you both! #modxbughunt

**Comment by pepebe** [comment]
I also prefer this version. I'll add them to the final PR. @ajilderda Thank!

**Comment by pepebe** [comment]
@ajilderda You code above solves the complete problem. do you want to do the PR? I can send you the modified css file.

**Comment by Mark-H** [comment]
Fixed in 2.6 per #13344
